packages/design-system/src/components/HelpDrawer/HelpDrawer.tsx

```diff
@@ -59,11 +59,11 @@ export const HelpDrawer = (props: HelpDrawerProps) => {
     });
 
     return () => {
-      // Send analytics event for helpdrawer open
+      // Send analytics event for helpdrawer close
       sendLinkEvent({
-        event_name: 'help_drawer_opened',
+        event_name: 'help_drawer_closed',
         event_type: EventCategory.UI_INTERACTION,
-        ga_eventAction: 'opened help drawer',
+        ga_eventAction: 'closed help drawer',
```

Comment:
I get a `"Drawer Toggle"` as the event on open now, and `"Close"` on close, seems like somethings wrong as I'm also getting some errors in console.. Will clear caches and see if maybe it's something local.

Reply:
Well clearing node_modules, building and restarting fixed my console errors but I still get the same `ga_eventLabel`'s

Reply:
It's working for me. There are actually two events that fire: one for the button clicks and one for the drawer opening/closing. It looks like you're only getting the button clicks.

Reply:
Looking deeper into how this could be happening for me, doesn't make sense as the code looks good.

Reply:
Ok I'm having a rough day, I was looking at Drawer, working great. 🚂